plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -145,6 +145,18 @@
       </consolePageParticipant>
    </extension>
    <extension
+         point="org.eclipse.ui.console.consolePatternMatchListeners">
+    <!-- pretty liberal URL pattern matcher -->
+    <consolePatternMatchListener
+      class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
+      id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
+      regex="https?://([\w-]+\.?)+(:\w+)?(/[^\s]*)?">
```

Comment:
I'm imagining a test that would extract the regex from the XML and compare it to strings.